tests/workspace_lint_config.rs

```diff
@@ -0,0 +1,26 @@
+use assert_cmd::Command;
+
+#[test]
+fn test_workspace_config() {
+    // TODO: we are only checking for required semver version overrides
+    // right now because that is all that is currently implemented.
+    let mut cmd = Command::cargo_bin("cargo-semver-checks")
+        .expect("cargo semver-checks command should exist");
+
+    cmd.current_dir("test_crates/manifest_tests/workspace_overrides")
+        .args([
+            "semver-checks",
+            "--baseline-root",
+            "old", // should be the workspace root, not package root
+            "--manifest-path",
+            "new/pkg/Cargo.toml",
+        ]);
+
+    let assert = cmd.assert();
+
+    // 1 minor: function_missing, overridden in workspace and not overridden in package
+    // 1 major: module_missing, overridden in workspace (minor) then overridden in package (major)
```

Comment:
Can you assert that these are indeed the minor and major checks that got triggered, and not e.g. vice versa? The current check could easily pass incorrectly, which would be a painful bug to track down.

In general, when you find yourself using a comment to describe something that should be true for a piece of code, assert it instead. Either the comment is accurate and the assert will pass, or it isn't and we need to know about it. The comment by itself risks being misleading: false, but pretending to be true. That tends to lead to lots of painful debugging.